GameServerCore/Packets/Interfaces/IPacketNotifier.cs

```diff
@@ -94,7 +94,9 @@ public interface IPacketNotifier
         void NotifySpawnStart(int userId);
         void NotifySpellAnimation(IAttackableUnit u, string animation);
         void NotifyStaticObjectSpawn(int userId, uint netId);
-        void NotifySurrender(IChampion starter, byte flag, byte yesVotes, byte noVotes, byte maxVotes, TeamId team, float timeOut);
+        void NotifySurrender(IChampion starter, bool open, bool votedYes, byte yesVotes, byte noVotes, byte maxVotes, float timeOut);
+        void NotifySurrenderStatus(int userId, TeamId team, SurrenderReason reason, byte yesVotes, byte noVotes);
+        void NotifyCanSurrender(bool can, TeamId team);
```

Comment:
Can should probably be renamed into something more descriptive.

Reply:
Often times when adding LeaguePackets we try to completely copy the name, with Notify at the beginning. I agree that the name is missing a bit.